### ISSUE TITLE
The 10-minute-tutorial does not work as expected

### ISSUE THREAD

**Issue by FatalCatharsis** [cucumber/docs#737]
### 👓 What did you see?

I was following along with the 10 minute tutorial [here](https://docs.cucumber.io/docs/guides/10-minute-tutorial/). Using maven, I downloaded the archtype project and then ran mvn test. The output showed nothing related to cucumber results. 

```

[INFO] Scanning for projects...
[INFO]
[INFO] --------------------< hellocucumber:hellocucumber >---------------------
[INFO] Building hellocucumber 1.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ hellocucumber ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory E:\projects\tests\hellocucumber\src\main\resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ hellocucumber ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ hellocucumber ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ hellocucumber ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ hellocucumber ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.116 s
[INFO] Finished at: 2022-04-05T12:50:17-05:00
[INFO] ------------------------------------------------------------------------
```

### ✅ What did you expect to see?

I expected this in the tests blocks:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running hellocucumber.RunCucumberTest
No features found at [classpath:hellocucumber]

0 Scenarios
0 Steps
0m0.004s

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.541 sec

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

### 📦 Which tool/library version are you using?

using maven 3.6.0 in command line and java 1.8.0-292 OpenJDK

### 🔬 How could we reproduce it?

Just follow the tutorial for maven.

### 📚 Any additional context?

----
 
*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/ISSUE_TEMPLATE/bug_report.md)*

**Comment by FatalCatharsis** [comment]
Note I do get normal junit test failures if I define an unimplemented feature in the src/resources/hellocucumber folder, but never recieve that report that is listed in the tutorial.

**Comment by mpkorstanje** [comment]
That's odd.

**Comment by mpkorstanje** [comment]
I would suggest updating the 10 minute tutorial and cucumber version:

* https://github.com/cucumber/docs/blob/main/content/docs/guides/10-minute-tutorial.md
* https://github.com/cucumber/docs/blob/main/data/versions.yaml

**Comment by bodiam** [comment]
I think the 10 minute tutorial is quite broken, in the way that it doesn't load the code examples anymore. 

I'm getting an error:

```
Uncaught TypeError: el is null
    addClass https://cucumber.io/js/site.js:17
    showOnly https://cucumber.io/js/site.js:73
    <anonymous> https://cucumber.io/js/site.js:140
```

This makes it very hard to figure out what to do.

**Comment by aurelien-reeves** [comment]
@bodiam I cannot reproduce your issue
Could you be more specific?
Did you tried hard-refreshing the page (using Ctrl+R for example)?

**Comment by bodiam** [comment]
@aurelien-reeves apologies, I should have included some screenshots:

<img width="899" alt="image" src="https://user-images.githubusercontent.com/46468/177167413-e7c39e0b-27ed-4cf9-9d53-c1faf20ba2cd.png">

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/46468/177167458-8665788d-fe32-4bc4-aa45-ba1e1a6d2dde.png">

<img width="613" alt="image" src="https://user-images.githubusercontent.com/46468/177167505-3bf2451a-9cc4-44c0-b316-31d86e2bbaf3.png">

I've tried this on both Chrome, Firefox and Safari on 2 different Macs, and I see no code examples at all, while the markdown file clearly has different code for each programming language.

**Comment by aurelien-reeves** [comment]
I still cannot reproduce your issue.

In your screenshot of your developer console, we can see that some content has been blocked. You may want to check your addons and configuratin to make sure you are not blocking some content from being downloaded.

I notice also a "SameSite" issue. What is the full URL you are using?

**Comment by bodiam** [comment]
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/46468/177181365-10df2398-1046-44d6-8eee-0ecf7f6df177.png">

This is the full URL I'm using: https://cucumber.io/docs/guides/10-minute-tutorial/. No content blockers, no addons, just plain Chrome

Could try reloading the page on your end? Maybe you have something cached which makes the page work?

**Comment by aurelien-reeves** [comment]
I did reload the page without reproducing the issue

However I have been able to reproduce it in incognito mode.

**Comment by aurelien-reeves** [comment]
@bodiam it should be fixed now. You just have to refresh the page.

Could you please confirm?

**Comment by bodiam** [comment]
<img width="974" alt="image" src="https://user-images.githubusercontent.com/46468/177192480-872aa9e4-c862-4cc2-8107-d89ba4cd48b7.png">

Success! Well done for fixing it, thanks!

**Comment by aurelien-reeves** [comment]
@FatalCatharsis @mpkorstanje has the initial issue been fixed? If so, could we close that ticket?

**Comment by epragtbeamtree** [comment]
@aurelien-reeves I don't think so, the 10 minute tutorial is still quite a bit out of date. I'm using it right now to setup a Kotlin/Gradle project, I'm happy to send a PR when I'm done with some updated documentation, such as the version numbers.

**Comment by aurelien-reeves** [comment]
Ok, thanks for the info

**Comment by mpkorstanje** [comment]
Unfortunately because of the latest JUnit 5 upgrade I've had to update the archetype to include a working test. This will  change the output a bit. This change should be released with Cucumber 7.5.0 and hopefully that will happen soonish.

https://github.com/cucumber/cucumber-jvm/pull/2590/files#diff-fa15bb75742d5701f909d046a514dbd9b99e704ffa5424c8a5228d910db91fcc

**Comment by epragtbeamtree** [comment]
I'm still happy to update the documentation, both for Maven and for Gradle. I'll try to do it before the release of Cucumber 7.5.0.

**Comment by mpkorstanje** [comment]
Please feel free. v7.5.0 has been released.

**Comment by luke-hill** [comment]
Hi @epragtbeamtree - Did you get around to this?

**Comment by bodiam** [comment]
@luke-hill  no, I never got around to doing this.
